src/renderer/services/plugin-manager.js

```diff
@@ -179,6 +179,9 @@ class PluginManager {
                 '$root',
                 '__vue__'
               ]
+              const badSetters = [
+                'innerHTML'
```

Comment:
We also need to add `outerHTML`

Reply:
I didn't know that was a thing 😂 I'll go through all getters to see what else there could be. Thanks 👌